.npmignore

```diff
@@ -0,0 +1 @@
+!schema.d.ts
```

Comment:
Maybe good to ignore the GH workflows as well I guess?

```suggestion
!schema.d.ts

.github
```

Reply:
Done.